src/Resources/public/Admin.js

```diff
@@ -154,6 +155,10 @@ var Admin = {
                   $(e.target).trigger('change');
               });
 
+            // In case some checkbox are already checked (for instance after a back via browser history), update iCheck checkboxes
```

Comment:
```suggestion
            // In case some checkboxes are already checked (for instance after moving back in the browser's session history), update iCheck checkboxes.
```